brainscore_vision/models/alexnet/test.py

```diff
@@ -2,12 +2,14 @@
 from pytest import approx
 
 from brainscore_vision import score
+from brainscore_vision.utils import seed_everything
+seed_everything(42)
```

Comment:
move inside methods where needed. `pytest` does not guarantee sequential test execution for instance.

But I also don't see what exactly we have to seed here. Everything should be deterministic:
* the model is uniquely specified
* the benchmarks have no randomness to them

So why do we need to seed?

Reply:
Seeding is necessary for models that generally use randomness during inference (e.g. untrained models with random initialization, VAEs, MC dropout), resulting in different scores even for the same benchmark run. I also think, generally, it's a good practice to seed test running for consistency. None of the models in this PR use it, so I removed seeding for this PR.